content/news/2024-??-??-bevy-0.15/index.md

```diff
@@ -33,7 +33,14 @@ Thank you so much to [everyone who helped out](https://discord.com/channels/6910
 The features above may be great, but what else does Bevy have in flight?
 Peering deep into the mists of time (predictions are _extra_ hard when your team is almost all volunteers!), we can see some exciting work taking shape:
 
-- TODO
+- **Bevy Scene Notation:** With required components, Cart's [master plan](https://github.com/bevyengine/bevy/discussions/14437) for BSN is starting to come together. Over the next few month, he's going to be heads-down developing a Bevy-specific file format (complete with matching macro and IDE support), the `Construct` trait (to easily include asset data in scenes), patches (to layer modifications to scenes) and experimenting with approaches to reactivity for UI.
```

Comment:
Rather than listing "item (to do thing)" I think it would make more sense to reverse it and say cart is experimenting with "approach to do thing (codenamed foo)".

Reply:
I thought BSN is "Bevy SceNe".